### DISCUSSION TITLE
ToggleButtonGroup

### DISCUSSION THREAD

**Discussion by smitev** [adobe/react-spectrum#3275] [Ideas]
I think it would be nice to have a ToggleButton Group with an option for exclusive and multiple selection.

Or it's possible to achieve the same functionality with a ButtonGroup maybe?

Inspiration:
[https://getbootstrap.com/docs/5.0/components/button-group/](https://getbootstrap.com/docs/5.0/components/button-group/)
[https://mui.com/material-ui/react-toggle-button/](https://mui.com/material-ui/react-toggle-button/)

**Comment by devongovett** [comment]
https://react-spectrum.adobe.com/react-spectrum/ActionGroup.html

**Reply by smitev** [reply]
I missed that one.
That's it.
Thanks.
